src/components/MultiSelect/AsyncMultiSelect.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { memo, useCallback, useMemo, VFC } from 'react';
+import { FormControl, FormLabel, FormErrorMessage } from '@chakra-ui/react';
+import { AsyncSelect } from 'chakra-react-select';
+import { useController } from 'react-hook-form';
+import { SelectOption } from '../../utils/selectOption';
+import { Debounce } from '../../utils/debounce';
+import { MultiSelect } from './base';
+
+interface Props extends MultiSelect.BaseProps {
+
+  /** Function for options loading. */
+  readonly fetchOptions: (searchString: string) => Promise<SelectOption[]>;
```

Comment:
Not sure if you redefining props from `chakra-react-select` here. If not, I think you should find a better name.